### PR TITLE
Add sum_of_numbers.js and reverse_integer.py

### PR DESCRIPTION
**Pull request by dephraiim**
Contributed for #16

### CODE REVIEW

**Comment by dephraiim** [comment]
@OtacilioN

**Comment by CaioCordeiro** [comment]
What a great code :smile: !
Just a little tip, its good to always put a description on your PRs to make more understandable.
Thank you for contributing! :rocket:
I would separate this PR in 3 PRs, one for the 'sum_of_numbers', other for the 'Strict equaling' fix and another for the 'reverse_integer' (I'll wait your response)

File: ReverseInteger/reverse_integer.py [hunk 1]
```diff
@@ -0,0 +1,9 @@
+def reverse_integer(integer):
+    def split(word): 
+        return [char for char in word] 
+
+    reversed = ""
+    return reversed.join(split(str(integer))[::-1])
+
+# Tests
+print(reverse_integer(43455))
```
Comment (CaioCordeiro) [inline ReverseInteger/reverse_integer.py:9]: Please add an empty line in the end of the file.

**Review by CaioCordeiro — CHANGES_REQUESTED** [review]

**Comment by OtacilioN** [comment]
Closing due to inactivity, feel free to reopen.

**Pull request closed without merging**